default/buildings.txt

```diff
@@ -999,6 +1026,7 @@ BuildingType
             Planet environment = Adequate
         ]
     ]
+    EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
```

Comment:
In general it is most clean & simple if you had made a new branch for your work (please do that next time)-- you would simply make the correction, commit it to your branch and then it would be automatically incorporated here.  Since it looks like you haven't yet merged anything new from the main master into your master branch on top of this, I think that might still be able to work-- so please try adding that new commit, but don't merge in anything new from master yet.